services/gateway/apis/http-version3/swagger/definitions/auth.json

```diff
@@ -0,0 +1,74 @@
+{
+    "authWithEnvelope": {
+        "type": "object",
+        "required": [
+            "data",
+            "meta",
+            "links"
```

Comment:
`links` is not mandatory in the response body and we do not send it in the response. Please correct it everywhere.
```suggestion
```